### DISCUSSION TITLE
"module 'fitz' has no attribute 'open'"

### DISCUSSION THREAD

**Discussion by RandomAnass** [pymupdf/PyMuPDF#1156] [Looking for help]
What happened to me is like what happened [here](https://github.com/pymupdf/PyMuPDF/issues/660) and [here](https://github.com/pymupdf/PyMuPDF/issues/129). But I couldn't solve the bug even by following the same instructions.

## Describe the bug (mandatory)
In my virtual environment I have this error :
`Error while analysing name.pdf : module 
'fitz' has no attribute 'open'     ` .
But I dont get this probelem outside the  virtual environment.

## To Reproduce (mandatory)
Outside the  virtual environment I tried :
`import fitz
fitz.open()
`
And I got : 
`Out[2]: Document(<new PDF, doc# 1>)`

To install the modules in my  virtual environment I used : 
`pip3 install -r requirements.txt`
And in requirements.txt : 

> fitz==0.0.1.dev2
>PyMuPDF==1.18.15
>PyPDF2==1.26.0

Then After the error I tried: 
`py -3 manage.py shell`
` >>>import fitz`
` >>>fitz.open()`
And I got:
`Traceback (most recent call last):
  File "<console>", line 1, in <module>
AttributeError: module 'fitz' has no attribute 'open'`

## My configuration (mandatory)
 - windows 10, 64 bits
 - Python 3.9.2
 - PyMuPDF version is 1.18.15

The output of `print(sys.version, "\n", sys.platform, "\n", fitz.__doc__)`  is : 3.9.2 (tags/v3.9.2:1a79785, Feb 19 2021, 13:44:55) [MSC v.1928 64 bit (AMD64)]
 win32
 None

for fitz I got:
 `fitz.__version__`

> Traceback (most recent call last):
>   File "<console>", line 1, in <module>
> AttributeError: module 'fitz' has no attribute '__version__'

**Comment by JorjMcKie** [comment]
There exists a package "fitz" on PyPI which you have installed and which has nothing to do with PyMuPDF.
You have somehow mixed up the environments containing "fitz" and "PyMuPDF" in a way I cannot judge from here.
The original creator of that package fitz used (parts of) his last name as the package name.

**Comment by JorjMcKie** [comment]
uninstall both completely and reinstall what you need of the two packages.
